### DISCUSSION TITLE
Concourse-pipeline-resource for 7.8.2

### DISCUSSION THREAD

**Discussion by ritik872000** [concourse/concourse#8502] [Help & Support]
So, I am trying to upgrade the concourse to 7.8.2 and using the latest image from the concourse-pipeline-resource  present in docker but it gives me error

fly version (6.3.0) is out of sync with the target (7.8.2). to sync up, run the following:

so, is it depreciated? or is there anything which i am mission out?

**Comment by matthope** [comment]
The "concourse-pipeline-resource" has been depreciated - https://github.com/concourse/concourse-pipeline-resource
